etcdserver/cluster.go

```diff
@@ -66,6 +67,9 @@ type Cluster struct {
 	// TODO: upgrade it as last modified index
 	index uint64
 
+	memberID  types.ID
+	transport rafthttp.Transporter
```

Comment:
// transport and members maintains the view of the cluster at index. This might be more upto date than what stores in the store since...